MTM-Frontend/src/pages/Auth/Register.tsx

```diff
@@ -97,13 +106,67 @@ const Register: React.FC = () => {
         values.password,
         values.userType,
       );
+
+      const user = {
+        password: values.password,
+        firstName: values.name.split(" ")[0],
+        lastName: values.name.split(" ")[1],
+        email: values.email,
+        phone: values.phone,
+        address: values.address,
+        city: values.city,
+        state: "state",
+        zip: parseInt(values.zip, 10),
+        role: "role",
+        household: "household",
```

Comment:
I think these fields are optional in database so you don't have to hard code this. Pls talk to Leo if things are unclear